local/local.go

```diff
@@ -102,7 +102,7 @@ func Locally(repo types.Repo, l types.Local, dry bool) bool {
 			}
 		} else {
 			auth = &http.BasicAuth{
-				Username: "xyz",
```

Comment:
@sjorsholtrop-ritense does bitbucket have an issue if it isn't the username?
GitHub has an issue with private repos. 
https://github.com/cooperspencer/gickup/discussions/313

I changed it back to `xyz` locally and it worked again.

Reply:
That's fine, we can 

I'll do some more testing as well, but at least github can clone with just a token. I will try that for codeberg, gitea, forgejo and onedev as well.

This was my quickfix on Saturday, https://github.com/cooperspencer/gickup/pull/314/files
I had to implement that for Gogs some time ago.
https://github.com/cooperspencer/gickup/blob/main/local/local.go#L99C11-L99C22

Basically setting the username as the token.

Reply:
I just tested it against Gitlab, Gitea, Codeberg and Onedev.
Gitea, Codeberg and Onedev are working fine with just the token.
Gitlab works fine with the Username and the token as password.

It is already in master.